### PR TITLE
Add category_id_start parameter to support starting category IDs from 1

### PR DESCRIPTION
**Pull request by xCatalitY**
This PR introduces a new parameter `category_id_start` to allow users to specify the starting value for category IDs when converting labelme annotations to COCO format. By default, the category IDs start from 0, but users can now set `category_id_start` to 1 or any other desired starting value.

Changes:
- Added `category_id_start` parameter to `get_coco_from_labelme_folder` function in `labelme2coco.py`
- Updated `convert` function in `__init__.py` to accept `category_id_start` parameter and pass it to `get_coco_from_labelme_folder`
- Modified `app` function in `cli.py` to include `category_id_start` parameter
- Updated README.md to document the usage of `category_id_start` parameter in both basic and advanced usage sections

Additionally, this PR fixes an issue where the file paths were incorrectly formatted in the "file_name" attribute when data is structured in subdirectories. Previously, if an image was located in a subfolder, e.g., `subfolder_1/image.jpeg`, the "file_name" attribute would only contain `image.jpeg` instead of the complete path. This has been fixed to include the full path in the "file_name" attribute.
This resolves #17

### CODE REVIEW

File: labelme2coco/cli.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 def app() -> None:
     """Cli app."""
-    fire.Fire(convert)
+    fire.Fire(lambda labelme_folder, export_dir="runs/labelme2coco/", train_split_rate=1, skip_labels=[], category_id_start=0: convert(labelme_folder, export_dir, train_split_rate, skip_labels, category_id_start))
```
Comment (fcakyon) [inline labelme2coco/cli.py:8]: This change is unnecessary. Once you revert this change , we can merge it 👍

File: labelme2coco/__init__.py
Comment (fcakyon) [inline labelme2coco/__init__.py:1]: Can you increment the version as well?

**Review by fcakyon — CHANGES_REQUESTED** [review]

File: labelme2coco/cli.py (hunk 1 above)
Comment (xCatalitY) [reply labelme2coco/cli.py]: Changed it back

File: labelme2coco/__init__.py
Comment (xCatalitY) [reply labelme2coco/__init__.py:1]: Done.

**Comment by xCatalitY** [comment]
Did your requested changes

**Review by fcakyon — APPROVED** [review]

**Comment by fcakyon** [comment]
@xCatalitY Thanks a lot for your contribution!
